editor/js/ui/editor.js

```diff
@@ -163,7 +163,7 @@ RED.editor = (function() {
     function validateNodeEditorProperty(node,defaults,property,prefix) {
         var input = $("#"+prefix+"-"+property);
         if (input.length > 0) {
-            if (!validateNodeProperty(node, defaults, property,input.val())) {
+            if (!validateNodeProperty(node, defaults, property,input.text())) {
```

Comment:
If the input is a `<select>` the change from `input.val()` to `input.text()` breaks here.